### PR TITLE
Added "List portal group users" sample

### PR DESCRIPTION
**Pull request by singhgag**

### CODE REVIEW

**Comment by maryharvey** [comment]
@singhgag 
- [ ] Image1.png is going to be public. Please confirm that Dan and Lucas are happy for these internal Avatar images to be made public. (small matter but worth checking).

File: arcgis-ios-sdk-samples/Cloud & Portal/List portal group users/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#List portal group users
+
+This sample demonstrates how to show users in a portal group.
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Cloud & Portal/List portal group users/README.md]: Add 'the' ...show the users

File: arcgis-ios-sdk-samples/Cloud & Portal/List portal group users/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+![](image1.png)
+
+##How it works
+
+The sample initializes an `AGSPortal` object. A group on this portal is retrieved using `findGroupsWithQueryParameters:completion:` method. From this group, all users are fetched using `fetchUsersWithCompletion:` method. The method returns an array of user names. An `AGSPortalUser` object is created for each user name using `initWithPortal:username:` initializer. Each of these portalUser object is loaded and the user information is shown in a table view.
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Cloud & Portal/List portal group users/README.md]: add 'the' 
....using the 'findGroupsWithQueryParameters:completion:' method
....using the 'fetchUsersWithCompletion:' method
....using the 'initWithPortal:username:' initializer
plural .... Each of these portalUser objects

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist [hunk 3]
```diff
@@ -651,6 +651,21 @@
...
+            <string>Cloud &amp; Portal</string>
+            <key>children</key>
+            <array>
+                <dict>
+                    <key>displayName</key>
+                    <string>List portal group users</string>
+                    <key>storyboardName</key>
+                    <string>GroupUsers</string>
+                    <key>descriptionText</key>
+                    <string>The sample demonstrates how to show users in a group</string>
```
Comment (maryharvey) [inline arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: add the .... show the users

**Comment by maryharvey** [comment]
@singhgag Perhaps a 'dummy' group should be set up with fictional characters so that the users description information is always valid.

**Comment by singhgag** [comment]
@maryharvey The group is created specially for the sample. Here is a [link](https://esri-runtime.slack.com/archives/sdk-samples/p1484777907000017) to the post on Slack. I will make the other changes, will take just a few minutes.

**Comment by maryharvey** [comment]
@singhgag I know the group was made for the sample but people might not realize that their image and text is being loaded up by runtime Samples as well as the image cut for the screenshot image. Just courtesy at least to ask.

**Comment by maryharvey** [comment]
@singhgag Just read the Slack message - fair enough. That's fine thanks - they all agreed to it.

**Comment by singhgag** [comment]
@maryharvey I have made THE other changes you suggested. ;)

**Pull request merged**